os/arch/arm/src/amebad/Make.defs

```diff
@@ -1,195 +1,199 @@
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common/drivers/wlan/realtek/include
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common/drivers/wlan/realtek/src/osdep
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common/api/platform
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common/api
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common/mbed/hal
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common/mbed/api
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common/mbed/hal_ext
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/common/mbed/targets/hal/rtl8721d
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/soc/realtek/amebad/misc
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/project/realtek_amebaD_va0_example/inc/inc_hp
-CFLAGS += -I$(TOPDIR)/board/$(CONFIG_ARCH_BOARD)/src/component/soc/realtek/amebad/cmsis
-
-CHIP_ASRCS  =
-CHIP_CSRCS = amebad_start.c
-CHIP_CSRCS += amebad_flash.c
-CHIP_CSRCS += amebad_idle.c
-CHIP_CSRCS += amebad_irq.c
-CHIP_CSRCS += amebad_timerisr.c
-CHIP_CSRCS += amebad_serial.c
-CHIP_CSRCS += amebad_lowerhalf.c
-CHIP_CSRCS += amebad_enet.c
-CHIP_CSRCS += amebad_timer_lowerhalf.c
-CHIP_CSRCS += amebad_rtc_lowerhalf.c
-CHIP_CSRCS += amebad_rtc.c
-CHIP_CSRCS += amebad_watchdog_lowerhalf.c
-ifeq ($(CONFIG_BOARDCTL_RESET),y)
-CHIP_CSRCS += amebad_systemreset.c
-endif
-ifeq ($(CONFIG_SYSTEM_REBOOT_REASON),y)
-CHIP_CSRCS += amebad_reboot_reason.c
-endif
-
-ifeq ($(CONFIG_AMEBAD_BLE),y)
-ifeq ($(CONFIG_ARCH_FPU),y)
-EXTRA_LIBS += ${TOPDIR}/board/rtl8721csm/src/libs/lib_btgap_fpu.a
-else
-EXTRA_LIBS += ${TOPDIR}/board/rtl8721csm/src/libs/lib_btgap.a
-endif
-endif
-
-ifeq ($(CONFIG_AMEBAD_TRUSTZONE),y)
-CHIP_CSRCS += amebad_nsc.c
-
-# trustzone library
-ifeq ($(CONFIG_ARCH_FPU),y)
-EXTRA_LIBS += ${TOPDIR}/board/rtl8721csm/src/libs/cmse_implib_fpu.a
-else
-EXTRA_LIBS += ${TOPDIR}/board/rtl8721csm/src/libs/cmse_implib.a
-endif
-endif
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+############################################################################
```

Comment:
I think you use different line end or something. Could you check whether this is made with linux format or not?

Reply:
Converted from linux to Win to resolve this issue.

Reply:
still, it is not resolved.
the file format is not correct because of different line end as sunghan said.
you can resolve it using `dos2unix` command.